.vscode/settings.json

```diff
@@ -11,6 +11,20 @@
 		"*.code-workspace": true,
 		"*.config.ts": true,
 		"coverage": true
+	},
```

Comment:
[i believe these should be set to false](https://github.com/utily/isly/pull/30#review-thread-or-comment-id-1210224260)